scripts/bundle.config.js

```diff
@@ -0,0 +1,81 @@
+/* global process */
+const {resolve} = require('path');
+const webpack = require('webpack');
+
+const getAliases = require('../aliases');
+
+const PACKAGE_ROOT = resolve('.');
+const PACKAGE_INFO = require(resolve(PACKAGE_ROOT, 'package.json'));
+
+/**
+ * peerDependencies are excluded using `externals`
+ * e.g. @deck.gl/core is not bundled with @deck.gl/geo-layers
+ */
+
+let externals = {};
+const {peerDependencies = {}} = PACKAGE_INFO;
+
+for (const depName in peerDependencies) {
```

Comment:
This automation looks very nice, though maybe create some functions with self documenting names, and add some one line comments to explain how the automation here works. Maintainers may not be as well versed with everything involved in this as you are right now.